<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsComposerVectorLegendBase</class>
 <widget class="QWidget" name="QgsComposerVectorLegendBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>419</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>7</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Vector Legend Options</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>10</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="textLabel1_2" >
     <property name="text" >
      <string>Title</string>
     </property>
     <property name="buddy" >
      <cstring>mTitleLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mTitleLineEdit" />
   </item>
   <item>
    <widget class="QLabel" name="textLabel1" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Map</string>
     </property>
     <property name="buddy" >
      <cstring>mMapComboBox</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="mMapComboBox" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mFontButton" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Font</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mFrameCheckBox" >
     <property name="text" >
      <string>Box</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Q3ListView" name="mLayersListView" >
     <column>
      <property name="text" >
       <string>Column 1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1_5" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Preview</string>
     </property>
     <property name="buddy" >
      <cstring>mPreviewModeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="mPreviewModeComboBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListView</class>
   <extends></extends>
   <header>q3listview.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mTitleLineEdit</tabstop>
  <tabstop>mMapComboBox</tabstop>
  <tabstop>mFontButton</tabstop>
  <tabstop>mFrameCheckBox</tabstop>
  <tabstop>mPreviewModeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
